### DISCUSSION TITLE
7.0.0 Beta

### DISCUSSION THREAD

**Discussion by cemrich** [mediathekview/zapp#287] [Announcements]
* Offizielle Unterstützung für Android TV!
* Neues Material 3 Design
* Android-Backup-Unterstützung für Einstellungen und Datenbankeinträge
* Kategorie-Icons heben sich im hellen Design vom Hintergrund ab
* Verbesserter Vollbild-Modus für Videos

<hr /><em>This discussion was created from the release <a href='https://github.com/mediathekview/zapp/releases/tag/7.0.0-beta'>7.0.0 Beta</a>.</em>

**Comment by ghost** [comment]
Leider gibt es noch bzw. wieder Probleme mit der Funktion "Im Hintergrund abspielen".

Das Problem mit dem Abbruch der Wiedergabe im Hintergrund war in einer früheren Version behoben worden. Allerdings habe ich mit v7.0.0-beta auch schon ein sofortiges Stoppen der Wiedergabe gehabt.
Kehre ich aber aus einer anderen App oder dem System Launcher in Zapp zurück, öffne genau den im Hintergrund laufenden Sender, bleibt teilweise die Steuerung eingeblendet. Ich muss dann den Bildschirm antippen oder die Wiedergabe anhalten und fortsetzen, um diese auszublenden. Das jedoch nur in Verbindung mit der Funktion "Im Hintergrund abspielen".
Jedoch bleibt die App nur für den Zeitraum im Vordergrund, den ich für das Bildschirm-Timeout eingestellt habe. Erst dunkelt der Bildschirm wie gewohnt ab und nach der eingestellten Zeit schaltet sich der Bildschirm ab und die Wiedergabe stoppt.
Wechsle ich in Zapp und hole nicht den im Hintergrund laufenden Sender in den Vordergrund, sondern wähle einen anderen Sender, läuft im Vordergrund alles korrekt weiter.
Diese Probleme sind nicht neu und bestanden auch schon in vorherigen Versionen.

Die Funktion "Bild im Bild" funktioniert problemlos wie sie soll, soweit ich das in der Kürze der Testzeit beobachten konnte.

**Reply by cemrich** [reply]
Vielen lieben Dank für's Testen und das Feedback!

**Reply by ghost** [reply]
Das ist das Mindeste, was ich für dich und diese prima App tun kann. **Vielen Dank für deine Arbeit!**

Leider sind nicht alle oben beschriebenen Fehler permanent nachvollziehbar.
Einen direkten Stopp der Wiedergabe nach der Rückkehr zu dem im Hintergrund abgespielten Sender hatte ich bisher nur einmal.
Zu den anderen Fehlern schreibe ich eventuell in den entsprechenden Threads _kurz_ etwas. Danke für deren Erstellung. 😉

**Comment by Moxitou** [comment]
Die app "zapp" funktioniert  gut, auf one (ard) ist der Ton häufig nicht synchron. Außerdem hängt das etwas duerftig geratene EPG mehrere Minuten hinter dem Ereignis hinerher.

**Comment by uragan1208** [comment]
Hallo,
besten Dank für das Bereitstellen der neuen Versionen.
kurzes Feedback von mir.
v6 -> läuft gut, hier stört aber der permanent eingeblendete rote "Info" Balken bei der Wiedegabe der Sender. Die Suche in der Mediathek lässt sich auf einem Xiaomi TV (Antroid TV) ohne angeschlossene Maus gar nicht durchführen.
v7 Beta -> Bei Starten der Anwendung dauernd ein Absturz.
Somit ist für mich die 3.2.0 immer noch die Version, die man am Leichtesten bedienen kann.

**Reply by cemrich** [reply]
Mit der gerade veröffentlichten Version 7.0.1 sollten die Abstürze auf älteren Versionen von Android TV behoben sein :)
